data-prepper-plugins/aggregate-processor/src/test/java/com/amazon/dataprepper/plugins/processor/aggregate/AggregateProcessorTest.java

```diff
@@ -157,7 +162,7 @@ void setup() {
         @Test
         void handleEvent_returning_with_no_event_does_not_add_event_to_records_out() {
             final AggregateProcessor objectUnderTest = createObjectUnderTest();
-            when(aggregateGroupManager.getGroupsToConclude()).thenReturn(Collections.emptyList());
+            when(aggregateGroupManager.getGroupsToConclude(eq(false))).thenReturn(Collections.emptyList());
```

Comment:
FYI, you don't need the `eq` here. You only need to specify an `ArgumentMatcher` when any of the parameters needs one. But, there is nothing wrong with keeping it here.